birdy/client/base.py

```diff
@@ -60,7 +61,9 @@ def __init__(
         """
         self._convert_objects = convert_objects
         self._converters = converters or copy(default_converters)
-
+        self._interactive = interactive
+        self._mode = ASYNC if interactive else SYNC
```

Comment:
not sure what will happen when we wait for a long-running process in sync mode.

Reply:
The same as for normal python statements ?

Reply:
Yes ... but the connection to the service may not last for long running processes. If we call the process async (when possible) we are on the safe side. From outside the function still waits because it is in the polling loop. 

In case of interactive mode we display the process (console message or widget).

Reply:
Correct. But if interactive is False, we don't want a polling loop.

Reply:
... but we might also use `birdy.client` as a library by other python modules? Well, for the demo it is fine :) We can decide later.